src/Common/ThreadStatus.h

```diff
@@ -153,27 +161,32 @@ class ThreadStatus : public boost::noncopyable
     /// Use it only from current thread
     ContextWeakPtr query_context;
 
-    String query_id;
+    String query_id_from_query_context;
 
     /// A logs queue used by TCPHandler to pass logs to a client
     InternalTextLogsQueueWeakPtr logs_queue_ptr;
 
     InternalProfileEventsQueueWeakPtr profile_queue_ptr;
 
+    struct TimePoint
+    {
+        void setUp();
+        void SetUp(std::chrono::time_point<std::chrono::system_clock> now);
```

Comment:
actually, it looks a bit strange to me : do we really need both `setUp` and `SetUp` (the last one in not in our code style)?